srml/contract/src/lib.rs

```diff
@@ -52,41 +52,14 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+// We need these `extern crate` to be placed here otherwise there will be errors.
+// TODO: https://github.com/paritytech/substrate/issues/1509
```

Comment:
What about leaving TODOs like these in code? (@gnunicorn )
Maybe we should add a direct reference to this code in the associated TODO.

Reply:
I think as far as it is linked it is good, but yeah good point that it should be linked back from the issue. I actually just forgot to do that : )

Reply:
Ah damn, it is already linked back, so I guess we are good now.